src/shared/ui/Icon/icon.ts

```
@@ -9,6 +9,7 @@ import IC_Error from '@/assets/icons/ic_error.svg';
 import IC_Like_Off from '@/assets/icons/ic_favorite_off.svg';
 import IC_Like_On from '@/assets/icons/ic_favorite_on.svg';
 import IC_Instagram from '@/assets/icons/ic_instagram.svg';
+import IC_Image_Folder from '@/assets/icons/ic_image_folder.svg';
```

ESLint

[error] Replace `mage_Folder·from·'@/assets/icons/ic_image_folder` with `nstagram·from·'@/assets/icons/ic_instagram`

(prettier/prettier)